test/iidm/NodeBreakerVoltageLevelTest.cpp

```diff
@@ -1046,6 +1152,15 @@ BOOST_AUTO_TEST_CASE(TerminalTest) {
     BOOST_CHECK_CLOSE(0, cBbs.getTerminal().getI(), std::numeric_limits<double>::epsilon());
 }
 
+BOOST_AUTO_TEST_CASE(TestRemoveVoltageLevelWithInternalConnectionsIssue) {
+    Network network("test", "test");
+
+    createNetwork(network);
+    network.getLine("L6").remove(); // needed to be allowed to remove the voltage level
+    network.remove(network.getVoltageLevel(S5_10K_V));
+    POWSYBL_ASSERT_THROW(network.getVoltageLevel(S5_10K_V), PowsyblException, "Unable to find to the identifiable 'S5 10kV'");
+}
```

Comment:
I think this test is also relevant for Bus/Breaker Voltage levels

Reply:
Ok added also In Bus/Breaker Voltage level tests.